src/main/java/io/cdap/plugin/gcp/bigtable/source/BigtableSource.java

```diff
@@ -146,6 +133,21 @@ public void transform(KeyValue<ImmutableBytesWritable, Result> input, Emitter<St
     }
   }
 
+  private Configuration getConfiguration() {
+    try {
+      Configuration conf = HBaseConfiguration.create();
+      BigtableConfiguration.configure(conf, config.getProject(), config.instance);
+      conf.set(BigtableOptionsFactory.BIGTABLE_RPC_TIMEOUT_MS_KEY, "60000");
```

Comment:
actually, there are a lot of advanced connection properties. I'm not sure end user needs some of them (see https://cloud.google.com/bigtable/docs/hbase-client/javadoc/com/google/cloud/bigtable/hbase/BigtableOptionsFactory). Maybe I should just remove this because timeouts are currently experimental. What do you think?

Reply:
let's remove this, but instead add a 'Bigtable Options' property that is just a map of key-values. Advanced users can add whatever they want there, which we will set in the Configuration.